INSTALL.rst

```diff
@@ -399,6 +399,9 @@ functionality is only available with the following installed:
 * `Dask <https://dask.org/>`__
     The ``dask`` module is used to speed up certain functions.
 
+* `Matplotlib <https://matplotlib.org>`__
+    Used in a number of the ``draw`` functions, for thresholding, segmentation
+    and as a plugin for reading images.
```

Comment:
Hmm, thanks for incorporating the feedback. Could you maybe rephrase further so that it doesn't sound like its necessary for all thresholding or segmentation functions?

Maybe something like 

```suggestion
    Used in a few functions to add interactivity, some segmentation functions
    as well as a plugin for reading images.
```

should stay up to date longer.. :crossed_fingers: 

As a followup idea, maybe we could modify the [`require` decorator](https://github.com/scikit-image/scikit-image/blob/c459c2adfc2b15e616c6858e034de178f4ba449a/skimage/_shared/version_requirements.py#L111) such that it adds a note to docstrings.

Reply:
I would keep it even more general, along these lines:

"Used in various functions, e.g., for drawing, segmenting, reading images."

> As a followup idea, maybe we could modify the [require decorator](https://github.com/scikit-image/scikit-image/blob/c459c2adfc2b15e616c6858e034de178f4ba449a/skimage/_shared/version_requirements.py#L111) such that it adds a note to docstrings.

Good idea!

Reply:
> As a followup idea, maybe we could modify the [require decorator](https://github.com/scikit-image/scikit-image/blob/c459c2adfc2b15e616c6858e034de178f4ba449a/skimage/_shared/version_requirements.py#L111) such that it adds a note to docstrings.

@lagru I wanted to create an issue with this content just so we don't forget. But, as I was trying to write the issue title/description, I realized I didn't know how to envision the change: Do we append a note to `name.__doc__`... on the fly? How does it get saved/printed? :thinking:

Reply:
Yes, modifying the `__doc__` is probably the best idea. Perhaps similar to [`_docstring_add_deprecated`](https://github.com/scikit-image/scikit-image/blob/eb35f51ac173f58dd1deee8c9ecc33efaa09af77/skimage/_shared/utils.py#L332).